### DISCUSSION TITLE
Docker on Doris: Latest fe and be docker image

### DISCUSSION THREAD

**Discussion by abhinav100499** [apache/doris#41173] [A - General / Q&A]
Which is the latest docker FE and BE individual image after 2.0.0_alpha? Need to perform the TCPH benchmarking [link](https://doris.apache.org/docs/benchmark/tpch)

**Comment by ixzc** [comment]
you can use more latest version such as 2.0.14 or 2.1.6, do not use 2.0.0_alpha

**Reply by abhinav100499** [reply]
Sure, but on docker hub not able to find the individual image for FE and BE after 2.0.0_alpha, also in docker compose examples were given using individual fe and be image.
Reference: https://github.com/apache/doris/blob/master/docker/runtime/docker-compose-demo/build-cluster/docker-compose/3fe_3be/docker-compose.yaml
Also, in example in doc used apache/doris:2.0.3-fe-x86_64 and apache/doris:2.0.3-be-x86_64 but not able to find on docker hub.

It would be great if you provide tag for the docker images for fe and be.
